server/game/cards/characters/06/ygritte.js

```diff
@@ -0,0 +1,20 @@
+const DrawCard = require('../../../drawcard.js');
+
+class Ygritte extends DrawCard {
+    setupCardAbilities(ability) {
+        // TODO: Cannot be knelt by card effects.
+        this.persistentEffect({
+            condition: () => this.controlsAnotherWildling(),
+            match: this,
+            effect: ability.effects.addKeyword('stealth')
+        });
+    }
+
+    controlsAnotherWildling() {
+        return this.controller.anyCardsInPlay(card => card !== this && card.getType() === 'character' && card.hasTrait('Wildling'));
```

Comment:
Kind of trying to get rid of `findCard` outside using it inside of `Player`, it's only a slight step of encapsulation above searching `cardsInPlay` directly. But I can change it.

Neither `findCard` nor this version of `anyCardsInPlay` searches attachments. I did continue off of this point on another branch to convert the rest of cards to use helper methods (no more direct usages of `.cardsInPlay`) and ended up changing `anyCardsInPlay` to search attachments. Doesn't really matter for Ygritte though.